.github/scripts/label_promoted_commits.py

```diff
@@ -77,9 +77,13 @@ def main():
                     comment = f'Closed via {commit.sha}'
                     add_comment_and_close_pr(pr, comment)
             else:
-                print(f'master branch, pr number is: {pr_number}')
-                pr = repo.get_pull(pr_number)
-                pr.add_to_labels('promoted-to-master')
+                try:
+                    pr = repo.get_pull(pr_number)
+                    pr.add_to_labels('promoted-to-master')
+                    print(f'master branch, pr number is: {pr_number}')
+                except:
```

Comment:
i'd suggest only catch the known exception(s), like:
```
from github import Github, GithubException
try:
    pr = repo.get_pull(pr_number)
    pr.add_to_labels('promoted-to-master')
    print(f'promoting pr#{pr_number} to master')
except GithubException.UnknownObjectException:
    print(f'{pr_number} is not a PR, no need to add label')
```
instead of swallowing all of them.